recipes/autoconf/all/conandata.yml

```
@@ -1,9 +1,9 @@
 sources:
   "2.72":
-    url: "https://ftpmirror.gnu.org/autoconf/autoconf-2.72.tar.xz"
+    url: "https://ftp.gnu.org/gnu/autoconf/autoconf-2.72.tar.xz"
```

```suggestion
    url:
      -  "https://ftpmirror.gnu.org/autoconf/autoconf-2.72.tar.xz"
      -  "https://ftp.gnu.org/gnu/autoconf/autoconf-2.72.tar.xz"
```
The reliability of ftpmirror is definitely a problem, but it should still be the first option since the download speed from the main site can be painfully slow, taking several minutes to download an archive. It can and should be added as a backup source instead.